app/templates/components/edit-reminder.hbs

```diff
@@ -13,7 +13,8 @@
       {{textarea class='new-reminder-notes' type='text' value=model.notes cols='80' rows='6'}}
     </label>
     {{#link-to 'reminders.reminder' model.id}}
-      <button type='submit' class='save-updates-btn'>Save</button>
+      <button class='save-updates-btn' {{action 'updateReminder' on='click'}}>Save</button>
```

Comment:
on click is the default button functionality - we dont need `on='click'` here. Probably applies to lots of other places too I might have missed that in earlier PRs